### PR TITLE
[@types/stellar-sdk] New types for stellar-sdk

### PR DESCRIPTION
**Pull request by Carl-Foster**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common-mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

This is my first time contributing to this project so if I've done something wrong, please tell me so I can fix it!

### CODE REVIEW

**Comment by Carl-Foster** [comment]
This adds types due to the requests at https://github.com/stellar/js-stellar-sdk/issues/132

File: types/stellar-sdk/tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (armanio123) [inline types/stellar-sdk/tslint.json]: Please don't change or disable the lint rules.

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/stellar-sdk/tslint.json (hunk 1 above)
Comment (Carl-Foster) [reply types/stellar-sdk/tslint.json]: Fixed

**Comment by Carl-Foster** [comment]
@armanio123 updated the lint rules

**Comment by Carl-Foster** [comment]
@typescript-bot addressed review comments

**Comment by nitzantomer** [comment]
@armanio123 
what's the status here?
the changes you asked for were fixed 3 days ago. thanks.

**Review by armanio123 — APPROVED** [review]

**Comment by armanio123** [comment]
I apologize it took some time. Your changes have been merged now.

**Pull request merged**